Helper/ScopeSetting.php

```diff
@@ -20,6 +20,8 @@ class ScopeSetting extends \Magento\Framework\App\Helper\AbstractHelper
     const KLAVIYO_PASSWORD = 'klaviyo_reclaim_user/klaviyo_user/password';
     const KLAVIYO_EMAIL = 'klaviyo_reclaim_user/klaviyo_user/email';
 
+    const KLAVIYO_OAUTH_NAME = 'klaviyo_reclaim_oauth/klaviyo_oauth/name';
```

Comment:
not required, but maybe:
```suggestion
    const KLAVIYO_OAUTH_NAME = 'klaviyo_reclaim_oauth/klaviyo_oauth/oauth_name';
```

Reply:
I am not a fan as now you repeat oauth the 3rd time.  can we get a second opinion, @smoucka ??

Reply:
maybe integration_name ? again not required but figured we can try and be descriptive if we can

Reply:
or consumer_name?

Reply:
IMO it's fine as is - you can see we follow a similar pattern for things like `klaviyo_reclaim_user/klaviyo_user/email`